app/assets/javascripts/event_emails.js.coffee

```diff
@@ -1,11 +1,30 @@
 if $('#select_recp').data('users') 
   $('#select_recp').prop("disabled", true)
 
-fill_email_to = (state) ->
-  $('#state'+state).click ->
-    users = $('#state'+state).data('users')
-    $('#event_email_to').val(users)
-    if users.length == 0
-      $('#event_email_to').attr('placeholder', 'No recipients present for ' + state + ' state')
-
-fill_email_to state for state in ['All', 'Accepted', 'Incomplete', 'Submitted', 'Approved', 'Cancel']
+# To remove the duplicate values from an array
+array_unique = (array) ->
+  unique_value = []
+  unique_value.push i for i in array when not (i in unique_value)
+  return unique_value
```

Comment:
There's probably some function or jquery model available for this. Did you check?